### PR TITLE
Add investors block

### LINKED ISSUE #481: [About Us] Investors

### PR DESCRIPTION
**Pull request by mdubus**
# Pull Request

## Related issue
Closes #481

## What does this PR do?
- Addition of new icons (investor's logos)
- Creation of the new `Investors` block
- Addition of the corresponding story in Storybook

Some more notes: 
- The `Lorem Ipsum` text is normal (to be updated later)
- I added another `Other block to come` separator to see the background-color differences between each block

## How to test

Go to [the deploy](https://landing-git-addinvestorblock-meili.vercel.app/about-us) and check the block
You can also `yarn storybook` and go to [the story](http://localhost:6006/?path=/story/blocks-investors--default)

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

File: data/about-us.js [hunk 1]
```diff
@@ -12,6 +22,48 @@ const aboutUsData = {
     title: 'Our story',
     image: '/images/timeline.png',
   },
+  investors: {
+    preTitle: 'Lorem Ipsum', // TODO : Update with text
```
Comment (bidoubiwa) [inline data/about-us.js:26]: Is this wanted?

File: src/blocks/Investors.js [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    margin-top: 96px;
+    padding: 0 88px;
+  }
+`
+
+const Investors = ({ investors }) => {
+  return (
+    <Section>
+      <TextSection>
+        <PreTitle color={get('colors.hotPink')} preTitle={investors.preTitle} />
```
Comment (bidoubiwa) [inline src/blocks/Investors.js:107]: PreTitle is used here but is a lorem ipsum text

File: data/about-us.js (hunk 1 above)
Comment (mdubus) [reply data/about-us.js:26]: Yes, I added a note on this on the PR body 🙈 

```
Some more notes:

- The Lorem Ipsum text is normal (to be updated later)
```

File: src/blocks/Investors.js (hunk 2 above)
Comment (mdubus) [reply src/blocks/Investors.js:107]: Yes I know it's normal for now, the text needs to be update

**Review by bidoubiwa — APPROVED** [review]

**Comment by mdubus** [comment]
bors merge

**Pull request merged**